### ISSUE TITLE
Inventory looks messy

### ISSUE THREAD

**Issue by mcapuccini** [kubenow/KubeNow#194]
Just did a test on CityCould (OpenStack). The inventory looks kind of messy:

```
[master]
clitest-master-000 ansible_ssh_host=37.153.138.137 ansible_ssh_user=ubuntu
[edge]

clitest-master-000 ansible_ssh_host=37.153.138.137 ansible_ssh_user=ubuntu
[master:vars]
nodes_count=4 
node_count=3
domain=37.153.138.137.nip.io
```

it should look something like

```
[master]
clitest-master-000 ansible_ssh_host=37.153.138.137 ansible_ssh_user=ubuntu

[edge]
clitest-master-000 ansible_ssh_host=37.153.138.137 ansible_ssh_user=ubuntu

[master:vars]
nodes_count=4 
domain=37.153.138.137.nip.io
```

@andersla could you fix it?

**Comment by mcapuccini** [comment]
p.s. @andersla I am really appreciating the master as edge feature 🙂

**Comment by andersla** [comment]
Yes I know:) Problem is limited functionality in terraform... looks like this due to workarounds, I might find another solution, but another option is that I would like to remove edges ip and hostnames from Inventory - they are only there for testing. Maybe we could accept not testing if they are in cloudflare register and only test that cheese.domain.etc is working.

**Comment by andersla** [comment]
The empy line is the zero-count edge hostname and ip, Terraform wont let you not output anything.

**Comment by mcapuccini** [comment]
@andersla it's not super important! I thought it was easyer to fix... but maybe:

```
[master]
clitest-master-000 ansible_ssh_host=37.153.138.137 ansible_ssh_user=ubuntu
[edge]
clitest-master-000 ansible_ssh_host=37.153.138.137 ansible_ssh_user=ubuntu
[master:vars]
nodes_count=4 
domain=37.153.138.137.nip.io
```

is this easyer to achieve? Also we really need both nodes_count and node_count?

**Comment by mcapuccini** [comment]
ping @andersla? Why do we need both node_count and nodes_count?

**Comment by andersla** [comment]
It was because of some testing playbook that wanted node_count (nodes - edges - master), I think I can remove it in a future pull request

**Comment by mcapuccini** [comment]
@andersla could you take care of this last issue so we can move toward release?

**Comment by mcapuccini** [comment]
fixed by #232
